internal/db/schema/migrations/oss/postgres/80/03_credential_static_base_table_updates.up.sql

```diff
@@ -7,19 +7,32 @@ begin;
   alter table credential_static add column create_time wt_timestamp;
   alter table credential_static add column update_time wt_timestamp;
```

Comment:
We could combine these two `alter table` commands into one, similarly to how `not null` is done lower done